mods/lord/Tools/lord_vessels/src/bottles.lua

```diff
@@ -33,3 +33,43 @@ minetest.register_node('lord_vessels:glass_bottle_water', {
 	groups          = { vessel = 1, dig_immediate = 3, attached_node = 1 },
 	sounds          = default.node_sound_glass_defaults(),
 })
+
+local glass_bottle_salt_texture = 'lord_vessels_salt_bottle.png'
+minetest.register_node('lord_vessels:glass_bottle_salt', {
+	description     = S('Glass Bottle (Salt)'),
+	drawtype        = 'plantlike',
+	tiles           = { glass_bottle_salt_texture },
+	inventory_image = glass_bottle_salt_texture,
+	wield_image     = glass_bottle_salt_texture,
+	paramtype       = 'light',
+	walkable        = false,
+	selection_box   = {
+		type  = 'fixed',
+		fixed = { -0.25, -0.5, -0.25, 0.25, 0.4, 0.25 }
+	},
+	groups          = { vessel = 1, dig_immediate = 3, attached_node = 1 },
+	sounds          = default.node_sound_glass_defaults(),
+})
+
+minetest.register_craft({
+	type = "cooking",
```

Comment:
please use single quotes everywhere

Reply:
Should I change them and commit?

Reply:
yes. Just new commit into the same branch